frontend/model/state.js

```diff
@@ -95,6 +98,13 @@ sbp('sbp/selectors/register', {
 // http://vuex.vuejs.org/en/mutations.html
 const mutations = {
   login (state, user) {
+    const monthTimestamp = currentMonthTimestamp()
```

Comment:
Could you had a comment of why you initialise the paymentsByMonth on login?

Reply:
done